requirements/coverage.txt

```diff
@@ -1,8 +1,36 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
-#    pip-compile requirements/coverage.in
+#    pip-compile --allow-unsafe requirements/coverage.in
 #
+build==0.8.0
+    # via pip-tools
+click==8.1.3
+    # via pip-tools
```

Comment:
The packages that're being added to the `requirements/*.txt` files are due to `pip-tools`, `pip-sync-faster` and their dependencies having been added.